AdyenCard/Components/Card/CardComponentExtensions.swift

```diff
@@ -21,11 +21,6 @@ extension CardComponent {
 
 extension CardComponent {
     
-    internal func isPublicKeyValid(key: String) -> Bool {
```

Comment:
No need to validate public key?

Reply:
nop, its fetched from the backend, right? no need IMO

Reply:
fair enough